google/cloud/contactcenterinsights/v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "contactcenterinsights.googleapis.com",
+    "api_shortname": "contactcenterinsights",
+    "client_documentation": "https://cloud.google.com/cpp/reference/contactcenterinsights/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/6143399",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Contact Center AI Insights API",
+    "product_documentation": "https://cloud.google.com/contact-center/insights/docs",
```

Comment:
nit: "https://cloud.google.com/solutions/ccai-insights" would be more consistent

Reply:
See above.